plugins/woocommerce/includes/class-wc-ajax.php

```diff
@@ -3075,6 +3075,11 @@ public static function shipping_classes_save_changes() {
 	 */
 	public static function toggle_gateway_enabled() {
 		if ( current_user_can( 'manage_woocommerce' ) && check_ajax_referer( 'woocommerce-toggle-payment-gateway-enabled', 'security' ) && isset( $_POST['gateway_id'] ) ) {
+			global $current_tab;
+
+			// Set current tab.
+			$current_tab = 'checkout';
```

Comment:
Minor, but could we check this from `$_SERVER['HTTP_REFERER']`?

`toggle_gateway_enabled` is not called from anywhere else at the moment, but this future proofs us in case we use it elsewhere in the future.

Reply:
Edit: probably better to check from `wp_get_referer()`.

Reply:
Good idea! I added this in the commit 780a703.